benchmark/Cabana_NeighborArborXPerformance.cpp

```diff
@@ -38,14 +38,14 @@ void performanceTest( std::ostream& stream, const std::string& test_prefix,
 
     // Declare problem sizes.
     double min_dist = 1.0;
-    std::vector<int> problem_sizes = { 1000, 10000, 100000, 1000000 };
+    std::vector<int> problem_sizes = { 100, 1000 };
```

Comment:
Are these taking too long?

Reply:
These were all set up mainly for running on Summit, so too long for CI. Maybe I should set up defaults for single CPU, single GPU, scaling etc.